src/gaurav_garg/PrintLastNumberAssignment12.java

```diff
@@ -0,0 +1,26 @@
+package gaurav_garg;
+
+public class PrintLastNumberAssignment12 {
+	
+	int lastnumber;
+	
+	void divide(int numb1 ,int numb2)
+	{	
+		
+		for(int i=numb1 ;i<=numb2 ;i++)
```

Comment:
you can iterate loop from end index as you need last number, so that you can get result in less number of iterations